map_data/geographical_region.txt

```diff
@@ -767,4 +768,98 @@ world_innovation_camel = {
 		world_kalimdor_tanaris
 	}
 }
-
+custom_sea_locked = {
+	regions = { 
+		world_eastern_kingdoms_kul_tiras
+		world_eastern_kingdoms_azeroth_stranglethorn
+		world_eastern_kingdoms_lordaeron_baradin_bay
+		world_northrend_howling_fjord
+		world_south_seas
+	}
+}
+custom_middle_kalimdor = {
+	regions = { 
+		world_kalimdor_azshara
+		world_kalimdor_barrens
+		world_kalimdor_durotar
+		world_kalimdor_dustwallow
+		world_kalimdor_mulgore
+		world_kalimdor_stonetalon
+		world_kalimdor_feralas
+		world_kalimdor_desolace
+		world_kalimdor_thousand_needles
+		world_kalimdor_uldum
+		world_kalimdor_silithus
+		world_kalimdor_tanaris
+	}
+}
+custom_desert_regions = {
+	regions = { 
+		world_eastern_kingdoms_khaz_modan_southern_khaz_modan
+		world_eastern_kingdoms_azeroth_redridge_mountains
+		world_kalimdor_barrens
+		world_kalimdor_durotar
+		world_kalimdor_desolace
+		world_kalimdor_thousand_needles
+		world_kalimdor_uldum
+		world_kalimdor_silithus
+		world_kalimdor_tanaris
+		world_south_seas_zandalar_voldun
+	}
+	duchies = {
+		d_edj
+		d_undermine
+	}
+}
+custom_taiga_regions = {
+	regions = { 
+		world_eastern_kingdoms_khaz_modan_dun_morogh
+		world_kalimdor_hyjal
+		world_northrend_borean_tundra
+		world_northrend_crystalsong
+		world_northrend_dragonblight
+		world_northrend_grizzly_hills
+		world_northrend_howling_fjord
+		world_northrend_icecrown_glacier
+		world_northrend_isle_of_conquest
+		world_northrend_sholazar_basin
+		world_northrend_storm_peaks
+	}
+}
+custom_jungle_regions = {
+	regions = { 
+		world_eastern_kingdoms_azeroth_stranglethorn
+		world_eastern_kingdoms_khaz_modan_wetlands
```

Comment:
Wetlands? It barely has trees if judge by WoW.

Reply:
Ya I was thinking about this one for a while, this wiki says its jungle though I remember it wasn't looking like one.

https://wowwiki-archive.fandom.com/wiki/Category:Jungles

Reply:
> Ya I was thinking about this one for a while, this wiki says its jungle though I remember it wasn't looking like one. But I agree, lmk if this wiki is wrong or I misunderstood.
> 
> https://wowwiki-archive.fandom.com/wiki/Category:Jungles

Don't use wowwiki.